build-tests/api-extractor-test-05/lib/index.d.ts

```diff
@@ -0,0 +1,20 @@
+/** @public */
+export namespace testJsLibrary {
+  /**
+   * This is my constant.
+   * @deprecated This is deprecated for a good reason.
+   */
+  export let MY_CONSTANT : number ;
+  /**
+   * This is my class.
+   */
+  export class MyClass {
+    private testJsLibrary_MyClass : any;
+    /**
+     * This method does something.
+     * @param x - This is x.
+     * @param y - This is y.
+     */
+    someMethod(x : number , y : string ) : boolean ;
+  }
+}
```

Comment:
Some of these names are a little odd (e.g. `testJsLibrary` and `testJsLibrary_MyClass`) - I know they're taken from our specific internal instance of this bug but maybe we can rename them something more generic here? Also, consider simplifying this test down to a truly minimal test case to make it clearer what we're exactly testing (maybe just an exported variable and exported function with extra spaces on both).

Reply:
I left in just enough to have multiple instances in the commits showing the error state for further verbosity, and figured that the additional 'bloat' would service to be more akin to a real scenario.

Minimizing this now.